table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GluePartition.java

```diff
@@ -45,4 +45,30 @@ public String getSpec() {
   public Layout getLayout() {
     return mLayout;
   }
+
+  @Override
+  public String toString() {
```

Comment:
Is there a way that `HivePartition` can also a string representation? Without it, we will only get useful information for only the Glue UDB, right?

Reply:
Add Hive `toString()` to get hive udb partition info.